subsys/nrf_rpc/CMakeLists.txt

```diff
@@ -18,4 +18,6 @@ zephyr_library_sources_ifdef(CONFIG_NRF_RPC_CALLBACK_PROXY nrf_rpc_cbkproxy.c)
 
 zephyr_library_sources_ifdef(CONFIG_NRF_RPC_UART_TRANSPORT nrf_rpc_uart.c)
 
-add_subdirectory_ifdef(CONFIG_NRF_RPC_DEV_INFO dev_info)
+if (DEFINED CONFIG_NRF_RPC_DEV_INFO OR DEFINED CONFIG_NRF_RPC_CRASH_GEN)
```

Comment:
no space between if and bracket in cmake

Reply:
Fixed